concept/type/entitytype.feature

```diff
@@ -1099,7 +1099,7 @@ Feature: Concept Entity Type
     When entity(person) set plays role: parentship:child
     When put entity type: man
     When entity(man) set supertype: person
-    When entity(man) set plays role: fathership:father as parent
+    When entity(man) set plays role: fathership:father as parentship:parent
```

Comment:
It was inconsistent to use unscoped role labels here, and scoped role labels elsewhere.